igcollect/linux_disklatency.py

```diff
@@ -107,5 +81,50 @@ def main():
             print(template.format(mode, k, int(v)))
 
 
+def get_path(args) -> str:
+    if args.path is not None:
+        return args.path
+
+    # no --path is specified -> use libvirt to get the storage volume
+    from libvirt import (
+        open as libvirt_open,
+        libvirtError,
+    )
+
+    if args.storagepool is None or args.storagevol is None:
+        print('you have to set --storagepool AND --storagevol')
+        sys.exit(1)
```

Comment:
`sys.exit`s should be just in the `main` function or it will get too confusing from where a script can be exited. Change it so the function is returning something or is raising an exception and the main function is handling it appropriately and doing the actual exit then.